src/Raven.Server/ServerWide/LowMemoryNotification/PosixLowMemoryNotification.cs

```diff
@@ -8,14 +8,15 @@
 using Raven.Server.Config;
 using Raven.Server.Config.Settings;
 using Voron.Platform.Posix;
+using Sparrow.Logging;
 
 namespace Raven.Server.ServerWide.LowMemoryNotification
 {
     public class PosixLowMemoryNotification : AbstractLowMemoryNotification
     {
         private readonly CancellationToken _shutdownNotification;
         private readonly RavenConfiguration _configuration;
-        private static readonly ILog Log = LogManager.GetLogger(typeof(PosixLowMemoryNotification));
+        private static readonly Logger _logger = _loggerSetup.GetLogger<PosixLowMemoryNotification>("PosixLowMemoryNotification");
```

Comment:
You cannot do this. You are relying on a field that is initialized in a method call to setup a field using static ctor.